README.md

```diff
@@ -8,19 +8,48 @@ Our open source community's Wiki focuses on answering your questions about learn
 
 ## Get a Fork ready
 
-You can get your own fork by using the button or clicking [this](https://github.com/FreeCodeCamp/wiki/new/master?readme=1#fork-destination-box).
-Once you get a fork, you will clone it with `git clone url` where `url` is the link to your fork.
+1. Fork it: You can get your own fork by using the button or clicking [this](https://github.com/FreeCodeCamp/wiki/new/master?readme=1#fork-destination-box).
```

Comment:
~~get your own fork~~ get your own fork of our wiki

Also: include `gif` of using the fork button.